roles/container-engine/containerd/tasks/main.yml

```diff
@@ -79,6 +79,15 @@
     mode: 0644
   notify: restart containerd
 
+- name: Ensure libseccomp is installed
+  package:
+    name: libseccomp
+    state: present
+  when:
+    - ansible_distribution in ["CentOS","RedHat","OracleLinux"]
```

Comment:
According to https://github.com/kubernetes-sigs/kubespray/pull/6922/files#r532726304 libseccomp package is not available on RHEL8. Is that due to the latest?
If we change here to present like this PR, is the package available on RHEL8?

Reply:
The message you are quoting says that it's already installed, which is normal since now systemd depends on it
chrony depends on it for a long time
```
# repoquery -q --installed --whatrequires libseccomp
Failed to set locale, defaulting to C.UTF-8
chrony-0:3.5-1.el8.x86_64
containerd.io-0:1.2.13-3.2.el7.x86_64
systemd-0:239-41.el8_3.x86_64
```

And to be complete, all recent containerd have the dependency, so we can just drop this for containerd and docker, sent a meesage on slack for crio
```
# repoquery -q --whatrequires libseccomp | grep -E 'containerd|docker'
Failed to set locale, defaulting to C.UTF-8
containerd.io-0:1.2.13-3.2.el7.x86_64
containerd.io-0:1.3.7-3.1.el7.x86_64
containerd.io-0:1.3.9-3.1.el7.x86_64
containerd.io-0:1.4.3-3.1.el7.x86_64
docker-ce-0:17.03.0.ce-1.el7.centos.x86_64
docker-ce-0:17.03.1.ce-1.el7.centos.x86_64
docker-ce-0:17.03.2.ce-1.el7.centos.x86_64
docker-ce-0:17.03.3.ce-1.el7.x86_64
docker-ce-0:17.06.0.ce-1.el7.centos.x86_64
docker-ce-0:17.06.1.ce-1.el7.centos.x86_64
docker-ce-0:17.06.2.ce-1.el7.centos.x86_64
docker-ce-0:17.09.0.ce-1.el7.centos.x86_64
docker-ce-0:17.09.1.ce-1.el7.centos.x86_64
docker-ce-0:17.12.0.ce-1.el7.centos.x86_64
docker-ce-0:17.12.1.ce-1.el7.centos.x86_64
docker-ce-0:18.03.0.ce-1.el7.centos.x86_64
docker-ce-0:18.03.1.ce-1.el7.centos.x86_64
docker-ce-0:18.06.0.ce-3.el7.x86_64
docker-ce-0:18.06.1.ce-3.el7.x86_64
docker-ce-0:18.06.2.ce-3.el7.x86_64
docker-ce-0:18.06.3.ce-3.el7.x86_64
docker-ce-3:18.09.0-3.el7.x86_64
docker-ce-3:18.09.1-3.el7.x86_64
docker-ce-3:18.09.2-3.el7.x86_64
docker-ce-3:18.09.3-3.el7.x86_64
docker-ce-3:18.09.4-3.el7.x86_64
docker-ce-3:18.09.5-3.el7.x86_64
docker-ce-3:18.09.6-3.el7.x86_64
docker-ce-3:18.09.7-3.el7.x86_64
docker-ce-3:18.09.8-3.el7.x86_64
docker-ce-3:18.09.9-3.el7.x86_64
docker-ce-3:19.03.0-3.el7.x86_64
docker-ce-3:19.03.1-3.el7.x86_64
docker-ce-3:19.03.10-3.el7.x86_64
docker-ce-3:19.03.11-3.el7.x86_64
docker-ce-3:19.03.12-3.el7.x86_64
docker-ce-3:19.03.13-3.el7.x86_64
docker-ce-3:19.03.14-3.el7.x86_64
docker-ce-3:19.03.2-3.el7.x86_64
docker-ce-3:19.03.3-3.el7.x86_64
docker-ce-3:19.03.4-3.el7.x86_64
docker-ce-3:19.03.5-3.el7.x86_64
docker-ce-3:19.03.6-3.el7.x86_64
docker-ce-3:19.03.7-3.el7.x86_64
docker-ce-3:19.03.8-3.el7.x86_64
docker-ce-3:19.03.9-3.el7.x86_64
docker-ce-3:20.10.0-3.el7.x86_64
docker-ce-3:20.10.1-3.el7.x86_64
```

Reply:
https://opensuse.pkgs.org/15.1/opensuse-update-oss-x86_64/cri-o-1.17.1-lp151.2.2.x86_64.rpm.html
https://opensuse.pkgs.org/tumbleweed/opensuse-oss-x86_64/cri-o-1.19.0-1.1.x86_64.rpm.html
The dependency is present for crio also

Reply:
libseccomp is a dependency since
https://centos.pkgs.org/7/docker-ce-stable-x86_64/containerd.io-1.2.13-3.2.el7.x86_64.rpm.html
And I just found out that there are finally EL8 builds
https://centos.pkgs.org/8/docker-ce-stable-x86_64/containerd.io-1.4.3-3.1.el8.x86_64.rpm.html

Reply:
Ok I'm removing the libseccomp install completely, let's see what the CI says